gulpfile.js

```diff
@@ -334,36 +319,37 @@ exports['publish:dts'] = function (done) {
 	return done();
 };
 
+class Subproject {
+	/**
+	 * @param {string} subprojectPath - E.g., "zul", "../zkcml/client-bind". With or without
+	 * a trailing slash ('/') doesn't matter, as we meticulously use `path.join`.
+	 */
+	constructor(subprojectPath) {
+		this.subprojectPath = subprojectPath;
+	}
+	get watcher() {
+		return () => watch_job(
+			path.join(this.subprojectPath, '**/*.ts'),
+			this.builder,
+		);
+	}
+	get builder() {
+		return () => typescript_dev_subproject(this.subprojectPath);
+	}
+}
+
+const subprojects = [
+	'zk',
+	'zul',
+	'../zkcml/zkex',
+	'../zkcml/zkmax',
+	'../zkcml/client-bind',
```

Comment:
Add more paths to this array to watch for more subprojects regarding browsersync. There are no restrictions for paths to the subprojects to end in a slash ('/') or not, as we meticulously use `path.join`.

Reply:
Can we check that these paths which contains with JS files can be added automatically?